bot.py

```
@@ -1,375 +1,64 @@
 
-# ===== Main Runner =====
 
 async def main():
     """
     Starts the Telegram bot and begins polling for updates asynchronously.
     """
-    logger.info("Starting bot")
+    logger.info("Starting bot ...")
+    try:
+        init_db()
+    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)